src/trading/NewOrdersFetcher.sol

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright (C) 2015 Forecast Foundation OU, full GPL notice in LICENSE
+ */
+
+pragma solidity ^0.4.13;
+
+import 'ROOT/Controller.sol';
+import 'ROOT/libraries/arrays/Bytes20Arrays.sol';
+import 'ROOT/libraries/Trading.sol';
+import 'ROOT/reporting/Interfaces.sol';
+import 'ROOT/trading/NewOrders.sol';
+
+
+/**
+ * @title OrdersFetcher
+ */
+// FIXME: This was named NewOrdersFetcher to prevent naming conflicts with ordersFetcher.se. Should be renamed to OrdersFetcher.
```

Comment:
Don't forget to apply this fixme before merging into develop.

Reply:
I haven't updated all of the other test scripts to use the Solidity version of Orders/OrdersFetcher (only test_orders.py), so that's why I was using this naming convention.  Should I go ahead and update the other test scripts as well before merging to develop?

Reply:
Yeah, this the goal for merging this PR is `orders.se` is deleted and `Orders.sol` fully replaces it (meaning all tests pass using Orders.sol instead of orders.se).